From 5303f366607b95d0c09d3d3f67e92ee59fc2760d Mon Sep 17 00:00:00 2001
From: Tomer Maimon <tmaimon77@gmail.com>
Date: Wed, 4 Sep 2024 18:35:24 +0300
Subject: [PATCH 22/23] arm64: dts: nuvoton: remove npcm750 compatible WD node
 name

Remove NPCM750 compatible WD node name since the WD driver using a unique
process for the NPCM845.

Signed-off-by: Tomer Maimon <tmaimon77@gmail.com>
---
 arch/arm64/boot/dts/nuvoton/nuvoton-common-npcm8xx.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/nuvoton/nuvoton-common-npcm8xx.dtsi b/arch/arm64/boot/dts/nuvoton/nuvoton-common-npcm8xx.dtsi
index 4a49ed86937c..ce450af5d13b 100644
--- a/arch/arm64/boot/dts/nuvoton/nuvoton-common-npcm8xx.dtsi
+++ b/arch/arm64/boot/dts/nuvoton/nuvoton-common-npcm8xx.dtsi
@@ -503,7 +503,7 @@ adc: adc@c000 {
 			};
 
 			watchdog0: watchdog@801c {
-				compatible = "nuvoton,npcm845-wdt", "nuvoton,npcm750-wdt";
+				compatible = "nuvoton,npcm845-wdt";
 				interrupts = <GIC_SPI 47 IRQ_TYPE_LEVEL_HIGH>;
 				reg = <0x801c 0x4>;
 				status = "disabled";
@@ -512,7 +512,7 @@ watchdog0: watchdog@801c {
 			};
 
 			watchdog1: watchdog@901c {
-				compatible = "nuvoton,npcm845-wdt", "nuvoton,npcm750-wdt";
+				compatible = "nuvoton,npcm845-wdt";
 				interrupts = <GIC_SPI 48 IRQ_TYPE_LEVEL_HIGH>;
 				reg = <0x901c 0x4>;
 				status = "disabled";
@@ -521,7 +521,7 @@ watchdog1: watchdog@901c {
 			};
 
 			watchdog2: watchdog@a01c {
-				compatible = "nuvoton,npcm845-wdt", "nuvoton,npcm750-wdt";
+				compatible = "nuvoton,npcm845-wdt";
 				interrupts = <GIC_SPI 49 IRQ_TYPE_LEVEL_HIGH>;
 				reg = <0xa01c 0x4>;
 				status = "disabled";
-- 
2.43.0

